### ISSUE TITLE
Filtering only works for files on the current page

### ISSUE THREAD

**Issue by brianharwell** [revenz/FileFlows#310]
#### What's it doing?
I have about 5000 files and I entered a search term on the Processed tab and nothing was displayed. I changed the page size to 5000 and did my search again and then I saw my files.

#### What should it do?
Search should return any matching files regardless of page size.

Version: 1.0.2.1951

**Comment by revenz** [comment]
no, that is a filter, for the data it currently has.   it is done on the client side.

Use the "Search" button to search the server side.

**Comment by revenz** [comment]
will make it so
1. if pager is not shown, filter is applied on client side
2. if pager is shown, the filter will execute on the server side.

**Comment by brianharwell** [comment]
Sounds great!!



Sent from myMail for iOS


Sunday, September 4, 2022, 6:24 PM -0400 from Timofey Biryukov  ***@***.***>:

**Comment by revenz** [comment]
updated so remove issue, now filter can now page the filtered data.

**Comment by brianharwell** [comment]
Thanks!

**Comment by brianharwell** [comment]
This looks much better but the debounce of 300ms to too short. As I type the screen keeps loading so it's hard to filter by multiple characters.

Maybe 750ms?

**Comment by revenz** [comment]
tried 750ms, found that too slow, increased to 500ms instead

**Comment by brianharwell** [comment]
500ms is much better.
